pkg/core/resources/apis/meshexternalservice/api/v1alpha1/meshexternalservice.go

```diff
@@ -118,8 +123,9 @@ const (
 
 type SANMatch struct {
 	// Type specifies matching type, one of `Exact`, `Prefix`. Default: `Exact`
+	// +kubebuilder:validation:Optional
 	// +kubebuilder:default=Exact
-	Type SANMatchType `json:"type,omitempty"`
+	Type SANMatchType `json:"type"`
```

Comment:
and here

Reply:
we agreed on slack to not add enums yet, will create an issue for this